packages/web/static/locales/en/faucet.json

```diff
@@ -35,9 +35,10 @@
   "testnetAddress": "Testnet Address",
   "getStarted": "Get Started",
   "done": "Done. SMS arriving \n in 1 – 2 Minutes",
-  "inviteError": "Unable to complete - double check your number",
-  "invalidNumber": "Must be a valid E.164 number",
+  "inviteError": "Unable to complete",
```

Comment:
im wondering if we should give more information here? 

at this point which changes to number input its very unlikely that the number is incorrect, maybe we should definitively say  something went wrong on our end?